client/packages/system/src/Immunisation/VaccineCourseView/VaccineCourseView.tsx

```diff
@@ -198,50 +138,64 @@ export const VaccineCourseView: FC = () => {
     const scheduleSeed = (number: number) => {
       return {
         id: FnUtils.generateUUID(),
-        number: number,
-        description: '',
-        day: 0,
+        doseNumber: number,
+        label: t('label.new-row'),
```

Comment:
The `new row` thing feels weird to me, having to remove it before I can put in the actual name of the dose - can it not just be blank? Or if you really want to put something here... "Dose number {num}"?